opencti-platform/opencti-front/src/private/components/analyses/malware_analyses/MalwareAnalysisCreation.tsx

```diff
@@ -196,7 +200,9 @@ MalwareAnalysisFormProps
   return (
     <Formik<MalwareAnalysisAddInput>
       initialValues={initialValues}
-      validationSchema={malwareAnalysisValidator}
+      validationSchema={validator}
+      validateOnChange={false} // Validation will occur on submission, required fields all have *'s
+      validateOnBlur={false} // Validation will occur on submission, required fields all have *'s
```

Comment:
Is it possible to extend these properties to all creation forms, so that they behave in the same way?

Reply:
That is the plan - there is a Slack conversation on it - iterative process :-) (I'll track down and link to you on Slack) . First step was to merge the PR - https://github.com/OpenCTI-Platform/opencti/pull/7437 - which has been done. Then, there was the stage to normalize the behavior like done here across creates/edits when these required fields PRs come over. Let me track down the Slack conversation on that and link it to you over on Slack

Reply:
Partially Slack - partially - here - https://github.com/OpenCTI-Platform/opencti/issues/7401

Reply:
Oh - I see in this PR - its not fully adopted to all creates/edits - :-/ - we can correct that.....